src/components/calcite-tab-title/calcite-tab-title.scss

```diff
@@ -1,124 +1,113 @@
-$tab-margin: 1.25rem;
-
 :host {
-  flex: 0 1 auto;
-  outline: none;
-  margin-right: $tab-margin;
-  margin-inline-start: 0;
-  margin-inline-end: $tab-margin;
+  @apply flex-initial outline-none mx-5;
+  margin-inline-start: theme("margin.0");
+  margin-inline-end: theme("margin.5");
 }
 
 :host([layout="center"]) {
+  @apply text-center my-0 mx-5;
   flex-basis: 200px;
-  text-align: center;
-  margin: 0 $tab-margin;
 }
 
 :host([position="below"]) a {
-  border-bottom: 0;
-  border-top: 3px solid transparent;
+  @apply border-b-0
+    border-t-2
+    border-t-color-transparent;
+  border-top-style: solid;
```

Comment:
would `border-solid` work here?

Reply:
Nope, if I remove `border-top-style: solid` here, and instead use `border-solid` in this apply directive, then when the calcite-tab-title's `position="below"`, the gray hover effect appears on all sides except for bottom